lib/ansible/modules/windows/win_acl.ps1

```diff
@@ -141,7 +168,9 @@ If (Test-Path -Path $path -PathType Leaf) {
 ElseIf ($inherit -eq "") {
     $inherit = "ContainerInherit, ObjectInherit"
 }
- 
+
+$ErrorActionPreference = "Stop" 
```

Comment:
I prefer a default $ErrorActionPreference "Stop", and then individual -ErrorAction "Continue" for each Cmdlet that we explicitly allow to continue. At least that way it is clear what the intention was by the developer. In all other cases we may inadvertently continue on specific errors...